Nasm.md

```diff
@@ -252,3 +252,68 @@ je .value3_case
 ```
 
 ## Data structures
+
+As you can imagine assembly doesn't have intrinsic concept of data structures, but luckily we have the support of the preprocessor where that let us declare struct like blocks.
```

Comment:
I would say this is the other way around: Every language supports accessing data as a raw array of bytes, C provides an abstraction over this in the form of structs. NASM also happens to provide us with an abstraction over raw bytes, that is similar to how C does it.